contracts/KashiPair.sol

```diff
@@ -678,8 +678,8 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
 
                 userCollateralShare[user] = userCollateralShare[user].sub(collateralShare);
                 userBorrowPart[user] = userBorrowPart[user].sub(borrowPart);
-                emit LogRemoveCollateral(user, address(this), collateralShare);
-                emit LogRepay(address(this), user, borrowAmount, borrowPart);
+                emit LogRemoveCollateral(user, swapper == ISwapper(0) ? to : address(swapper), collateralShare);
```

Comment:
yes, these streamlines the `from` & `to` to actually match the expectation of the things that happened . Correct me if I'm wrong here.